src/extension/iSamples_resultList.js

```diff
@@ -152,4 +152,4 @@ ResultList.propTypes = {
   children: PropTypes.array
 };
 
-export default React.memo(ResultList);
+export default ResultList;
```

Comment:
Remove unnecessary HOC. React memo is only for the class components that have no state parameters and depend only on the prop parameters.